be/src/exec/line_reader.h

```diff
@@ -24,7 +24,7 @@ namespace doris {
 namespace io {
 class IOContext;
 }
-// This class is used for CSV scanner, to read content line by line
```

Comment:
add docs for enclose and escape.

Reply:
I'll make it when when it is sure to be correct